test/safe/branch.test.ts

```diff
@@ -14,6 +14,16 @@ describe("Branch Webhook", () => {
 	let app: Application;
 	const gitHubInstallationId = 1234;
 
+	const mockGitHubAuthRequest = () => {
+		githubNock.post(`/app/installations/${gitHubInstallationId}/access_tokens`)
```

Comment:
Optionally is missing

Reply:
I did that on purpose.  Optionally is frowned upon with nock unless it's outside the scope of your tests.  This particular call is important and we want our test to fail if it's missing from the code.